ush/ufsda/stage.py

```diff
@@ -12,7 +12,8 @@
 import logging
 import glob
 import numpy as np
-from pygw.yaml_file import YAMLFile
+import yaml
```

Comment:
is `import yaml` used at all?

Reply:
You're right!    `pygw.yaml_file` replaces `yaml`.   `import yaml` removed at [dc5db4a](https://github.com/RussTreadon-NOAA/GDASApp/commit/dc5db4a0106930f835008f69dfc38db36b2fa7cf).